ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh

```diff
@@ -62,7 +62,7 @@ snyk_code() {
     fi
 
     if [ -z "${TARGET_REFERENCE}" ]; then
-        TARGET_REFERENCE=$(echo $CLONE_REFS | jq -r .base_ref)
+        TARGET_REFERENCE=$(echo $CLONE_REFS | jq -r .base_ref | tr '.' '-')
     fi
     PARAMS+=(--target-reference="${TARGET_REFERENCE}")
```

Comment:
In case `TARGET_REFERENCE` exists already, the proposed substitution won't take place. Consider applying outside of the `if` block.